index.html

```diff
@@ -1,237 +1,242 @@
-  <section class="contact-section ">
-    <div class="contact-content">
-            <div class="contact-me">
-              <h2>contact me</h2>
+              <ul class="social-media-icon">
+                <li><a class="social-media" href="#"><img src="images/github.png" alt="github"></a></li>
+                <li> <a class="social-media" href="#"><img src="./images/linkedin.png" alt="linkedin"></a> </li>
+                <li><a class="social-media" href="#"><img src="images/anglelist.png" alt="anglelist"></a> </li>
+                <li><a class="social-media" href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+                <li><a class="social-media" href="#"><img src="images/medium.png" alt="medium"></a></li>
+              </ul>
+            </div>
+            <div class="middle-sec">
+              <div class="btn-divb">
+                <button class="btn" type="submit">Get my resume</button>
+              </div>
+            </div>
+            <ul>
+              <ul class="chev">
+                <li class="p-lan">Languages </li>
+                <li><i class="fa fa-chevron-down"></i></li>
+              </ul>
+              <ul class="img">
+                <li><img class="lan-g" src="images/Frame 9.png" alt="Javascript"></li>
+                <li><img class="lan-g" src="images/Frame 10.png" alt=" HTML"></li>
+                <li><img class="lan-g" src="images/Frame 11.png" alt="CSS"></li>
+              </ul>
+              <ul class="chev">
+                <li class="p-lan">Frameworks </li>
+                <li><i class="fa fa-chevron-right"></i></li>
+              </ul>
+              <hr class="line">
+              <ul class="chev">
+                <li class="p-lan">Skills</li>
+                <li><i class="fa fa-chevron-right"></i> </li>
+              </ul>
+              <hr class="line">
+            </ul>
+          </div>
+          </div>
+          </div>
+        </section>
+      </section>
+      <!-- ====================================================contact form ======================================== -->
+      <section class="contact-section bg">
+        <div class="contact-content">
+          <div class="contact-me">
+            <h2>contact me</h2>
           </div>
           <div class="about-us">
-                <p>
-                  If you have an application you are interested in developing, 
-                  a feature that you need built or a project that needs coding. 
-                  I’d love to help with it!
-                </p>
+            <p>
+              If you have an application you are interested in developing,
+              a feature that you need built or a project that needs coding.
+              I’d love to help with it!
+            </p>
           </div>
-      
           <form action="https://formspree.io/f/{form_id}" method="post">
             <div>
               <input type="text" name="userName" maxlength="30" placeholder="userName" required>
```

Comment:
Good job working on the contact form 😃. but you still ned to fix some parts

- [ ] Currently, we are getting an error when we send a message. Kindly make sure you fixed it 

![Capture1](https://user-images.githubusercontent.com/57408419/230064475-a053dbdb-ba13-4664-8c5c-a55dcb785e8c.JPG)

- [ ] Also make sure you stuck to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃
  - Kindly make sure you fixed your `border-top-left-radius`
  - Kindly make sure you removed the `border` on the `input`  to look like [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture](https://user-images.githubusercontent.com/57408419/230066558-6e409f2b-883a-46f7-8f30-d0d798723b9b.JPG)| |![Capture0](https://user-images.githubusercontent.com/57408419/230066616-7acd7b41-dba8-4191-97f7-92106e71d703.JPG)| |